### DISCUSSION TITLE
Emulating video cards.

### DISCUSSION THREAD

**Discussion by WhiteMagicRaven** [86Box/86Box#3008] [General]
Hello.
I am want to get understand basics on how to emulate video card(s).

I trying to collect video cards and my dream to emulate them. 

For example Rendition Verite as it has own Graphic API, and games that require it, without card it will not work or work in software mode which is lower quality than with card.

From where to start =) or if someone doing something, how i can help, i remember previously submitted all the bioses here.

**Comment by Torinde** [comment]
Related:
- #1933
